README.md

```diff
@@ -109,6 +103,124 @@ from `run_benchmark()` (both default `FALSE`):
   If `TRUE`, the result data will contain a `prof_file` field, which you can 
   read in with `profvis::profvis(prof_input = file)`. 
 
+## Defining benchmarks
+
+Benchmarks are constructed by `Benchmark()`, which takes expressions that handle 
+setup, teardown, and the actual work that we want to measure. See its 
+documentation for details, and see `read_file` and `write_file` for examples.
+
+### A (contrived) example
```

Comment:
Rather than adding a contrived version of the `read_csv` benchmark, what if you added these extensive comments to the actual benchmark and pointed to that from here? That way there's less risk that our detailed, documented example becomes stale.

Reply:
Yeah, I had thought about doing that, but I worry that some of the uniqueness in that benchmark (and honestly each of them have their own quirks) would distract more than it would help. I will go back and comments to those benchmarks as well.